pkg/util/cloudproviders/cloudfoundry/cccache.go

```diff
@@ -898,4 +898,5 @@ func (ccc *CCCache) reset() {
 	ccc.orgQuotasByGUID = make(map[string]*CFOrgQuota)
 	ccc.processesByAppGUID = make(map[string][]*cfclient.Process)
 	ccc.cfApplicationsByGUID = make(map[string]*CFApplication)
+	ccc.lastUpdated = time.Time{}
```

Comment:
What do you think about adding a comment in the code that this is only used for testing?

Reply:
It's not the case anymore after this [PR](https://github.com/DataDog/datadog-agent/pull/13555/files#diff-93609910d8d587655a550e93aa0d93385550606d29b1071f7b841cbb175d7501).

Reply:
Ah sorry for the confusion- I was referring to the `reset()` function as well.